package.json

```
@@ -33,7 +33,7 @@
     "fetch"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=12"
```

```suggestion
    "node": "^12.20.0 || ^14.14.0 || >=16.0.0"
```

To match https://github.com/netlify/cli/blob/f560c82885df081c94b53ae2e3db1b6e4bf839da/package.json#L14

---

I thought about it too, but this package seems to work with Node.js >= 12.

Feel free to change it if you think it's worth it, I tend to limit engines based on what's really needed :)

---

Oh, BTW we should also change the TS Config file. What's the ES equivalent for Node.js 12.x?

---

All right, I think this should be OK now. I used `ES2019` and your engines suggestion, although arguably Node.js >=12 works too in this project.

---

I believe those are the minimal versions required to support ES Modules without an additional flag.
At some point we'd want to migrate this lib to ES modules

---

Yeah, that's true.